portal/src/texts/blog/cursor-pointer.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "Å peke, eller ikke peke"
+author: Jo Emil Holen
+description: Et valg mellom å følge operativsystem eller norm
+publishDate: "2021.09.28"
+---
+
+<Ingress>
+    Operativsystemene og browser defaults tilsier at det kun er linker som skal ha en egen musepeker, men normen på
+    internett er at alle klikkbare elemenenter har dette satt. Hvordan angriper man en sånn problemstilling for å ta et
+    konsekvent valg?
+</Ingress>
+
+Om du trykker rundt i operativsystemet på datamaskinen din vil du se at når du fører musepekeren over en ting er det veldig sjeldent du får en egen musepeker. Dette er stort sett forbeholdt linker, og sånn har det vært ment helt siden vi så den spesielle pekeren i [Apple sitt HyperCard i 1987](https://adamsilver.io/blog/buttons-shouldnt-have-a-hand-cursor-part-2/).
+
+I designretningslinjenene til de forskjellige operativsystemene kan vi lese at en [musepeker av typen "pointer" indikerer en link](https://www.w3.org/TR/CSS21/ui.html#propdef-cursor).
+Det samme ser vi på et ustilet HTML-element. Hverken knapper eller select har en egen musepeker ved hover som standard.
+
+## Men om vi ser på web er saken en helt annen.
+
+```css
+button {
+    pointer: cursor;
+}
+```
+
+Her har knapper, lenker, selects og andre klikkbare elementer mer eller mindre konsekvent en egen peker.
+Både på større og mindre nettsteder ser man dette, det samme med designsystemer som [Carbon](https://www.carbondesignsystem.com/components/select/usage/) og [Material You](https://mui.com/components/buttons/).
+
+Hvorfor det har blitt sånn er vanskelig å si. Kanskje det er en missforståelse om at klikkbare elementer skal ha en egen peker. Eller kanskje man bare har fortsatt med det fordi "alle andre har det" gjennom historien. Kanskje det kommer fra et tredjeparts bibliotek.
```

Comment:
Skulle vi hatt spørsmålstegn i stedet for punktum på disse kanskje-setningene?
For øvrig én "s" i misforståelse.

Reply:
Jeg er litt usikker på spørsmålstegn. Det var tenkt mer som påstander enn spørsmål, men da er kanskje setningene litt feil?

Reply:
Funker sikkert greit, tenker begge varianter er OK.